webpack/test_setup.js

```diff
@@ -20,3 +21,4 @@ jest.mock('./assets/javascripts/foreman_tools', () => ({
   foremanUrl: url => url,
 }));
 jest.mock('./assets/javascripts/foreman_navigation');
+console.log(new Date());
```

Comment:
Debug left over?

Reply:
its in the [test the tests] commit, all js tests stopped working over the weekend with time out, this pr passed last week so I'm using it to check